pkg/minishift/provisioner/minishift_detector_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioner
+
+import (
+	"testing"
+	"reflect"
+	"github.com/docker/machine/libmachine/provision"
+	"github.com/minishift/minishift/pkg/minikube/tests"
+	"github.com/docker/machine/libmachine/drivers"
+)
+
+func TestMinishiftDetector(t *testing.T){
```

Comment:
Maybe we should change the test name as well, this test is really about making sure the minishift provisioner is selected - _TestMinishiftProvisionerSelected_